DB/common.php

```diff
@@ -885,6 +885,9 @@ function autoPrepare($table, $table_fields, $mode = DB_AUTOQUERY_INSERT,
     function autoExecute($table, $fields_values, $mode = DB_AUTOQUERY_INSERT,
                          $where = false)
     {
+        if ($where) {
+            $where = strtr($where, array('?' => '\?', '!' => '\!', '&' => '\&',));
```

Comment:
Can you expand your testcase such that all three of those characters are present in the WHERE?

Reply:
Read #3